tests/kube_ctl/watch_pods_test.go

```diff
@@ -0,0 +1,29 @@
+package kube_ctl
```

Comment:
what was this test about?

Reply:
At the time, I was thinking of coverage for kubectl `WatchPods()` and `WatchPodsInNs()` to ensure OS specific commands are issued to `RunVPanic`(...).  If you still think this would be good to have, I can add it to my list.  If not, we can remove this test setup.